app/code/Magento/Inventory/Setup/InstallData.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Setup;
```

Comment:
@maghamed 
If I am not mistaken we decide to put this functionality in bridge module (which @bartekszymanski introduced on Contribution Day)

Reply:
@naydav  good catch! 

yeah, that should be placed in InventoryCatalog module